server/routes/api/documents.test.ts

```diff
@@ -2492,3 +2492,84 @@ describe("#documents.unpublish", () => {
     expect(res.status).toEqual(401);
   });
 });
+
+describe("#documents.empty_trash", () => {
+  it("should require admin", async () => {
+    const viewer = await buildViewer();
+    const res = await server.post("/api/documents.empty_trash", {
+      body: {
+        token: viewer.getJwtToken(),
+      },
+    });
+    expect(res.status).toEqual(403);
+  });
+
+  it("should require authentication", async () => {
+    const res = await server.post("/api/documents.empty_trash");
+    expect(res.status).toEqual(401);
+  });
+
+  it("should permenantly delete trashed documents", async () => {
+    const { admin, document } = await seed();
+    // To simulate deleting draft
+    document.publishedAt = null;
+    await document.save();
+    await document.delete(admin.id);
```

Comment:
Context - I couldn't figure out why drafts weren't getting deleted. Turns out it was `Document.findAll` being scoped to `publishedAt: [Op.ne]: null`. Since we already require `admin` privileges, I kept `findAll` _unscoped_ for `empty_trash` endpoint .